target/linux/ipq806x/files-4.19/arch/arm/boot/dts/qcom-ipq8065.dtsi

```diff
@@ -75,4 +75,12 @@
 			};
 		};
 	};
+	
+	cpus {
```

Comment:
Shouldn't this part go into separate patch? I quite miss how this is related to the changes in qcom-ipq8064.dtsi

Reply:
It's related to cpuidle. Should I make 2 commit, one for cpuidle enable and one for disabling spc on ipq8065?

Reply:
In order to make it clear, yes.